src/main/charts/bitbucket/values.yaml

```diff
@@ -567,6 +562,17 @@ bitbucket:
       #
       passwordSecretKey: password
 
+  shutdown:
+    # -- The termination grace period for pods during shutdown. This
+    # should be set to the Bitbucket internal grace period (default 30
+    # seconds), plus a small buffer to allow the JVM to fully terminate.
+    terminationGracePeriodSeconds: 35
+
+    # By default, Bitbucket will be stopped by the script supplied by the image.
+    # If any other shutdown behaviour is needed, it can be achieved by overriding this value.
+    command:
+      - "/shutdown-wait.sh"
```

Comment:
Similar to the `fluentd.command` should we not just default to the `/shutdown-wait.sh` script in the respective helper function, meaning it doesn't need to be explicitly defined here? Looking at the helper for this there isnt actually any need to supply this value here
```
{{/*
The command that will be run to properly shutdown Bitbucket
*/}}
{{- define "bitbucket.shutdown.command" -}}
{{- if .Values.bitbucket.shutdown.command }}
{{- .Values.bitbucket.shutdown.command }}
{{- else }}["/shutdown-wait.sh"]{{- end }}
{{- end }}

```

Reply:
We provide many of the default values in the `values.yaml`. To me having the command blank in that context implies no command is run by default, which is not true. We should probably be consistent about this, or add a note about what the default is and where it is set.